test/row.js

```diff
@@ -447,7 +447,7 @@ describe('Bigtable/Row', function() {
 
     var families = [
       {
-        name: 'test-family',
+        id: 'test-family',
```

Comment:
This should be reverted so that we can do a separate PR for families in ReadRows